### PR TITLE
Item Class & console startup

### LINKED ISSUE #1: [1.5pt] Implement Methods
1. can_be_archived?() in the Item class
     - should return true if published_date is older than 10 years
    - otherwise, it should return false
2. move_to_archive() in the Item class
    - should reuse can_be_archived?() method
    - should change the archived property to true if the result of the can_be_archived?() method is true
    - should do nothing if the result of the can_be_archived?() method is false

### LINKED ISSUE #4: [1.0pt] All Item class properties visible in the diagram should be defined and set up in the constructor method.
Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.

### LINKED ISSUE #5: [1.0pt] Create Item class in a separate .rb file.
For each class you should create separate `.rb` file

### LINKED ISSUE #6: [0.5pt] Add all methods visible in the diagram
You should add all methods which are visible in the diagram

### LINKED ISSUE #7: [1.0pt] Create a main.rb file that will serve as your console app entry-point.
- [ ] Create `main.rb` file which should serve as console app entry point

### LINKED ISSUE #8: [1.0pt] Implement startup actions
1. Present the user with a list of options to perform.
2. Let users choose an option.
3. If needed, ask for parameters for the option.
4. Have a way to quit the app.

### PR DESCRIPTION
**Pull request by ZahraArshia**
### requirements:

- [x] All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.
- [x] Implement startup actions:
    - Present the user with a list of options to perform.
    - Let users choose an option.
    - If needed, ask for parameters for the option.
    - Have a way to quit the app.

** guys I have a linter error that I couldn't fix 😢

### CODE REVIEW

File: classes/startup.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    2. List all musics
+    3. List of games
+    4. Add a book
+    5. Add a music
+    6. Add a game
+    7. Exit"
+    gets.chomp
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```
Comment (bushmusi) [inline classes/startup.rb]: @ZahraArshia you have done great on switching the case. In my opinion we have to try to solve linter error instead of disable

**Review by bushmusi — APPROVED** [review]
Hi @ZahraArshia 

You have done a great job 👍 , 

- [x] No linter error
- [x] OOD has been followed
- [x] DRY and clean code

File: classes/startup.rb (hunk 1 above)
Comment (ZahraArshia) [reply classes/startup.rb]: Thank you @bushmusi to check it, I'm open to suggestions about it

Comment (bushmusi) [reply classes/startup.rb]: @ZahraArshia I made changes to the above issue. Now without disable linter we can proceed

Comment (bushmusi) [reply classes/startup.rb]: @ZahraArshia now our startup console is ready. It is time to merge it

**Pull request merged**